CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Breaking Changes
+  * Make `PayPalRequest` and subclasses `Parcelable`
```

Comment:
Updating `PayPalCheckoutRequest` to implement `Parcelable` shouldn't break existing integrations with `PayPalCheckoutRequest`, right? I wouldn't think this is a breaking change.

We should also call out the full list of classes that got updated in this PR

Reply:
The concern here was anyone subclassing any of these classes would have to implement `Parcelable` on their subclass  now too.  There shouldn't be a need to subclass this class in order to integrate, but should it still be considered a breaking change because of that?

Reply:
Hm - I would think it is unlikely that a merchant would be subclassing the requests nor is it our intended use-case, so I would opt to say this isn't a breaking change. That is my vote, but trust whatever y'all decide!

Reply:
Bumped it up to a regular change!